lmod_proxy/edx_grades/actions.py

```diff
@@ -119,3 +129,13 @@ def get_sections(gradebook, form):
         data,
         not bool(error_message)
     )
+
+
+def _get_assignment_name_max_points(form):
+    try:
+        csv_data = form.datafile.data.stream.read()
+        short_name = csv_data[0][1]
```

Comment:
It looks like you're reading the column header from column 2 but you are reading the column value from column 3 below. Is that correct?